app/components/works/detail_component.rb

```diff
@@ -104,6 +104,22 @@ def rejection_reason
       work.last_rejection_description
     end
 
+    def previous_user_versions?
+      return false unless Settings.user_versions_ui_enabled
+
+      return true if work_version.user_version > 1
```

Comment:
L110-112 could just be `work_version.user_version > 1`

Reply:
Fixed, thanks.